cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, {Component, PropTypes} from 'react';
+import MyRuleEngineApi from 'api/rulesengine';
+import enableDataPreparationService from 'components/DataPrep/DataPrepServiceControl/ServiceEnablerUtilities';
+import LoadingSVG from 'components/LoadingSVG';
+import T from 'i18n-react';
+
+require('./RulesEngineServiceControl.scss');
+const PREFIX = 'features.RulesEngine.RulesEngineServiceControl';
+
+export default class RulesEngineServiceControl extends Component {
+
+  static propTypes = {
+    onServiceStart: PropTypes.func
+  };
+
+  state = {
+    loading: false
+  };
+
+  enableRulesEngine = () => {
+    this.setState({
+      loading: true
+    });
+    enableDataPreparationService({
+      shouldStopService: false,
+      artifactName: 'rules-engine-service',
+      api: MyRuleEngineApi,
+      i18nPrefix: ''
```

Comment:
Shouldn't be. The only place where i18nprefix is used is during version mismatch in dataprep (have a min version of dataprep). It won't happen for rules engine as we are just releasing it in 4.3. When we have some hard constraints on yare service versioning then we can add i18n and can pass on the prefix.